lib/src/components/NxTextInput/NxTextInput.tsx

```diff
@@ -114,6 +114,11 @@ const NxTextInput = forwardRef<HTMLDivElement, Props>(
     }
 );
 
-NxTextInput.propTypes = propTypes;
+const PublicNxTextInput = forwardRef<HTMLDivElement, PublicProps>(function(props, ref) {
```

Comment:
This part really should have been done previously, since NxTextInput already had separate PublicProps